NSS-Day-Cohort-49/tabloid-full-stack-spicy-sweet-chilis
# Pr setup

#### Changes Made
1. Added files for `Comments` 
?
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout pr-setup
    ```
2. Open a new Terminal tab (?T) and navigate to the server directory.
3. Test app functionality.
    > login and navigate to the comments tab. once logged in, because this feature is dependent on the postId which Sue just implemented, you have to manually enter a postId at the end of the url and then re-render the page to see the results. let me know if you have any questions. 
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
